### ISSUE TITLE
`elevation` fxn: changes needed probably

### ISSUE THREAD

**Issue by sckott** [ropensci/rgbif#344]
have seen some usage of this fxn in papers, so think carefully about this

https://developers.google.com/maps/documentation/elevation/usage-and-billing Changing it seems to require a credit card, and pricing I think is going up. 

maybe change to using a free option? geonames? give options of what to use?

**Comment by sckott** [comment]
using the sample data

```r
library(rgbif)
occ_key <- name_suggest('Puma concolor')$key[1]
dat <- occ_search(taxonKey = occ_key, return = 'data', limit = 300,
    hasCoordinate = TRUE)
```

current version on CRAN using Google elevation API:

```r
install.packages("rgbif")
library(rgbif)
elevation(latitude = dat$decimalLatitude[1:10],
  longitude = dat$decimalLongitude[1:10],
  key=getOption("g_elevation_api"))

   latitude longitude elevation
1  32.90944 -116.5459 1443.2424
2  38.54031 -122.9700  327.3882
3  37.05063 -121.4302  337.8897
4  31.91320 -109.9317 1515.2610
5  32.64865 -108.5365 1761.6854
6  32.64997 -108.5372 1768.9541
7  47.42263 -113.6708 1227.2605
8  30.06013 -103.5581 1354.2333
9  37.04786 -121.4257  346.3244
10 40.36340 -120.6227 1295.2610
```

dev version on the `elevation-change` branch - using Geonames API:

```r
remotes::install_github("ropensci/rgbif@elevation-change")
library(rgbif)
elevation(latitude = dat$decimalLatitude[1:10],
      longitude = dat$decimalLongitude[1:10],
      username = getOption("geonamesUsername"))

   latitude longitude elevation
1  32.90944 -116.5459      1447
2  38.54031 -122.9700       340
3  37.05063 -121.4302       340
4  31.91320 -109.9317      1524
5  32.64865 -108.5365      1769
6  32.64997 -108.5372      1780
7  47.42263 -113.6708      1230
8  30.06013 -103.5581      1353
9  37.04786 -121.4257       373
10 40.36340 -120.6227      1297
```

Elevation values are slightly different. Does it matter? Anyone have thoughts?

**Comment by sckott** [comment]
fixed in #345
